src/de/jost_net/JVerein/gui/menu/KursteilnehmerMenu.java

```diff
@@ -34,6 +36,8 @@ public class KursteilnehmerMenu extends ContextMenu
    */
   public KursteilnehmerMenu(TablePart table)
   {
+    addItem(new CheckedSingleContextMenuItem("Bearbeiten", new KursteilnehmerDetailAction(),
+        "text-x-generic.png"));
```

Comment:
Sollten wir in Zeile 43 "Zum Mitglied machen" in "Zu Mitglied umwandeln" so wie der Text auch in Nicht-Mitglied Menü lautet.

Reply:
Mache ich
Hier ist es momentan so, dass ein neues Mitglied mit den Daten des Kursteilnehmers erstellt wir und dieser nicht gelöscht wird. Entweder müsste man hier löschen oder den Eintrag anders nennen.

Reply:
Dann würde ich es anders nennen z.B. "Neu als Mitglied"

Reply:
Was hältst  du von "Zu Mitglied übernehmen"?

Reply:
auch ok

Reply:
habe ich geändert